Dockerfile-alpine.template

```diff
@@ -0,0 +1,83 @@
+FROM php:7.1-%%VARIANT%%
+
+LABEL maintainer="pierre@piwik.org"
+
+RUN set -ex; \
+	\
+	apk add --no-cache --virtual .build-deps \
+		alpine-sdk \
```

Comment:
Doh sorry, wish I'd noticed this sooner -- this is basically the same as `build-essential` in Debian.  If you're not building Alpine APK packages, this is the wrong package to install.  You probably want instead `gcc`, `make`, etc directly instead (but also, `docker-php-ext-configure` will install those, so perhaps you wanted instead to simply include `$PHPIZE_DEPS` here?)

Reply:
This seems to be required for the pecl extensions.

Reply:
This is more than is required, is the point I'm making -- it should be `$PHPIZE_DEPS` instead (which is the set of packages the image scripts install for doing the same thing)

Reply:
Thanks, got it. This is fixed in #100.